### PR TITLE
Add media production

### PR DESCRIPTION
**Pull request by statik**
Opening a PR to host the conversation

### CODE REVIEW

**Comment by statik** [comment]
@crashGoBoom today @knmurphy and I worked on testing out NICE DCV instead of parsec, and it went really well! What would you think about making NICE DCV the base ami that we start from? https://aws.amazon.com/marketplace/seller-profile?id=74eff437-1315-4130-8b04-27da3fa01de1
https://aws.amazon.com/hpc/dcv/

File: aws/cdk/mediaproduction/cdk.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "app": "npx ts-node bin/mediaproduction.ts",
+  "context": {
+    "@aws-cdk/core:enableStackNameDuplicates": "true",
+    "aws-cdk:enableDiffNoFail": "true"
```
Comment (statik) [inline aws/cdk/mediaproduction/cdk.json:5]: oooh, neat!

**Comment by crashGoBoom** [comment]
Just pushed some changes I never actually committed earlier. Havent tested this much since we moved on though.

**Review by statik — COMMENTED** [review]
merging so we don't lose this

**Pull request merged**